activity_test.go

```diff
@@ -116,3 +118,40 @@ func TestActivityUnmarshallBadPayloadTo(t *testing.T) {
 		t.Fatal("To payload was not a value feedslug:userid format, expected To to be nil afterward, got:", activity.To)
 	}
 }
+
+func TestActivityUnmarshalScore(t *testing.T) {
+	activity := &getstream.Activity{}
+
+	expected := 1.123
+	payload := []byte(fmt.Sprintf(`{ "score": %f }`, expected))
+
+	err := activity.UnmarshalJSON(payload)
+	if err != nil {
```

Comment:
Indeed, but I wanted to use a separate branch just for cleaning up tests (including using testify).